src/shared/email/templates/password-change-notification.html

```
@@ -0,0 +1,293 @@
+            .title {
+                font-size: 18px;
+            }
+            .info-row {
+                flex-direction: column;
+                align-items: flex-start;
+            }
+            .info-value {
+                margin-top: 4px;
+            }
+        }
+    </style>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </style> ]

(tag-pair)